.github/workflows/build.yaml

```diff
@@ -1,6 +1,5 @@
-name: build
+name: build_packages
```

Comment:
Do we need to update the status badge in the README to look for the new name instead?

Reply:
Looking at the badge it's only based on the filename that didn't change.

Reply:
I'll do a follow up to add the test file too.